src/python/pants/backend/debian/register.py

```diff
@@ -0,0 +1,13 @@
+# Copyright 2021 Pants project contributors (see CONTRIBUTORS.md).
```

Comment:
At least for now, we want this `register.py` to live at `backend/experimental/debian/register.py`. That way, you activate `pants.backend.experimental.debian`, and we manage expectations for users that this is still experimental. 

The code itself can still live in `backend/debian`, and then you'd import it in this file. Only the `register.py` matters for the backend's name.